src/main/flight/pid.c

```diff
@@ -413,7 +413,12 @@ static float calcHorizonRateMagnitude(void)
 static void pidLevel(pidState_t *pidState, flight_dynamics_index_t axis, float horizonRateMagnitude)
 {
     // This is ROLL/PITCH, run ANGLE/HORIZON controllers
-    const float angleTarget = pidRcCommandToAngle(rcCommand[axis], pidProfile()->max_angle_inclination[axis]);
+    float angleTarget = pidRcCommandToAngle(rcCommand[axis], pidProfile()->max_angle_inclination[axis]);
+
+    // Automatically pitch down if the throttle is manually controlled and reduced bellow cruise throttle
+    if ((axis == FD_PITCH) && STATE(FIXED_WING) && FLIGHT_MODE(ANGLE_MODE) && !navigationIsControllingThrottle())
+        angleTarget += scaleRange(MAX(0, navConfig()->fw.cruise_throttle - rcCommand[THROTTLE]), 0, navConfig()->fw.cruise_throttle - PWM_RANGE_MIN, 0, mixerConfig()->fwMinThrottleDownPitchAngle);
```

Comment:
I think we should start thinking about unifying configuration for navigation and other things into "platform configuration". We can have separate `airplaneConfig` and `rotorcraftConfig` plus unified `platformConfig` to keep common values. These structures will hold platform capabilities like max inclination, cruise throttle, hover throttle, reference airspeed etc.

Reply:
:+1:

Reply:
It's definitely out of scope of current PR, but we should really look into it.